.travis.yml

```diff
@@ -13,9 +13,11 @@ before_script:
 - ./node_modules/.bin/eslint assets/js/components/*.js
 - ./node_modules/.bin/eslint assets/js/constants/*.js
 - # ./node_modules/.bin/eslint assets/js/controllers/*.js
+- ./node_modules/.bin/eslint assets/js/controllers/e*.js
 - # ./node_modules/.bin/eslint assets/js/core/*.js
 - # ./node_modules/.bin/eslint assets/js/directives/*.js
 - ./node_modules/.bin/eslint assets/js/filters/*.js
+- ./node_modules/.bin/eslint assets/js/helpers/*.js
```

Comment:
Not really, services usually provide the functionality of helpers. @Lyncee59 could you rename please?